eng/pipelines/templates/steps/build-test.yml

```diff
@@ -50,6 +50,7 @@ steps:
         --service="${{ parameters.ServiceDirectory }}"
         --toxenv="${{ parameters.ToxTestEnv }}"
         --injected-packages="${{ parameters.InjectedPackages }}"
+        --package-name="{{ parameters.BuildTargetingString }}"
```

Comment:
I suspect it might be more flexible to just pass the name of the coverage file down instead of passing the package name. Especially given you use that full file name in other places it would make it easier to keep things in sync, and there will be less coupling in your script.